app/api/sessions.py

```diff
@@ -51,8 +52,9 @@ def after_create_object(self, session, data, view_kwargs):
             organizer = session.event.get_organizer()
             organizer_email = organizer.email
             frontend_url = get_settings()['frontend_url']
+            concerned_event = EventModel.query.filter(Event.id == session.event_id).first()
```

Comment:
No need, you already have session.event

Reply:
@iamareebjamal Understood, will use it in next commit